development/image-syncer/external-images.yaml

```diff
@@ -55,4 +55,6 @@ images:
   amd64Only: true
 - source: "ghcr.io/kedacore/keda:2.10.1"
 - source: "ghcr.io/kedacore/keda-admission-webhooks:2.10.1"
-- source: "ghcr.io/kedacore/keda-metrics-apiserver:2.10.1"
\ No newline at end of file
+- source: "ghcr.io/kedacore/keda-metrics-apiserver:2.10.1"
+- source: "nginx:1.23.3"
+  amd64Only: true
```

Comment:
Why do we want to only sync AMD64 image?

Reply:
nginx would have docker images for different OS. I saw [this](https://github.com/kyma-project/test-infra/pull/7740/files#diff-1d1e45de46cf46126a18450e766d3211dabb8a8ee454f4411f3e989cb31a4d46L52) so thought it would only sync specific images.